webpack.config.js

```diff
@@ -98,16 +98,12 @@ module.exports = (env, argv) => {
         logo: `${sourceDir}/images/favicon.png`,
         prefix: "images/favicons/",
         favicons: {
-          appName: "Saleor ecommerce",
-          appDescription: "Store front for the Saloer ecommerce platform",
+          appName: "Saleor",
+          appDescription: "Store front for the Saleor ecommerce platform",
```

Comment:
There are two other mistakes in this sentence:
- "Store front" -> "Storefront"
- "ecommerce" -> "e-commerce"